lib/chef/provider/batch.rb

```diff
@@ -27,11 +27,7 @@ class Batch < Chef::Provider::WindowsScript
       def command
         interpreter_path = Chef::Util::PathHelper.join(basepath, interpreter)
 
-        "\"#{interpreter_path}\" #{flags} \"#{script_file.path}\""
-      end
-
-      def flags
-        new_resource.flags.nil? ? "/c" : new_resource.flags + " /c"
+        "\"#{interpreter_path}\" #{new_resource.flags} /c \"#{script_file.path}\""
```

Comment:
Do the extra spaces we end up with here hurt us if flags is nil?

Reply:
It doesn't seem to affect it.